### DISCUSSION TITLE
how to extract beam information from MatRad

### DISCUSSION THREAD

**Discussion by Mariam950-wp** [e0404/matRad#619] [Q&A]
Hi, 

I need to access the optimized beam parameters in matrad after simulating proton beam interaction with CShape geometry. 

Is it possible?

**Comment by wahln** [comment]
I don't really understand what beam parameters exactly you are interested in.
As data structures, you can find the geometrical positions of the proton spots in the `stf`, the dose information per spot in the `dij` and the optimized intensities in `resultGUI`. Check the Wiki on our GitHub page for more technical information on these data structures.

**Comment by Mariam950-wp** [comment]
Thanks for your reply,

I was wondering the geometry of the proton beam with which optimized dose distribution was achieved?

is this information accessible in MatRad?

thanks,
Mariam Abuladze

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Mariam950-wp** [comment]
more precisely,

I would like to know if it is possible to extact the parameters (energy, gantry angle, weight) of the proton beam with which the best dose distribution was achieved for the specific phantom.

Thanks
Mariam Abuladze
________________________________

**Comment by Mariam950-wp** [comment]
Now I am trying to do it for TG119
________________________________
